### ISSUE TITLE
Support for Animation skeleton bones

### ISSUE THREAD

**Issue by ksadgum** [FlaxEngine/FlaxEngine#2638]
It would be really useful if It was possible to animate skeletons like you would in blender.

**Comment by xxSeys1** [comment]
Nothing against this and I think this would be very useful too, but why exactly do u need this functionality in the engine? What do you think animation tooling in Flax could do what Blender or Mayas can not? 

Even big engines like Unity and Unreal (have) struggle(-d) to make good animation tools so it's unlikely that the tools Flax will provide will be as good as theirs or for example Blenders or Mayas.

This sounds more like a workflow issue and the root prolly lies more at the import workflow (both from the user and engine side) than missing tooling in the engine.
Also it's a ton of scope and I'm sure Mafi and contributors have more important stuff to work on.

Again I'm not directly against it, just trying to better understand why you want this feature.

**Comment by ksadgum** [comment]
Yeah of course there are more important stuff, but the reason I need it is to make slight changes to the animations in case I messed some part of them in blender (instead of going back to blender, import and export...), and to add other animations that you only know how to make once you are in the middle of making the game (like hand controls when interacting with others, carrying objects,...). Of course you can just make base animations in blender and brute force them into every event in the game.
Another thing you mentioned is Unity and Unreal. I wasn't actually comparing Flax to neither of those two but rather to Godot. Flax is far better than Godot when it comes to 3d, but yet godot has built in skeleton bones animations (not as good as blender, but it get the job done, and that's what I meant).
But again, there are more important things devs should focus on.
